### ISSUE TITLE
CSV export argument must be str not bytes

### ISSUE THREAD

**Issue by thomastx05** [wakatime/wakadump#8]
Anytime i try to run this is get the above error and only an empty file is create.

**Comment by alanhamlett** [comment]
Should be fixed in v3.0.1.

**Comment by thomastx05** [comment]
Yes, I got it to work yesterday sorry. I just changes the unicodecsv py3.py file to make it work.

Having said that…  the file that comes out is not very helpful.  Would it possible to output a file such as the attached? This would be much more useful as one could create pivot tables to group and dissect the data a lot better in this format.

**Comment by thomastx05** [comment]
I wrote a little console app that spits them back to me the way I need. You are welcome to use if you would like (it is nothing fancy). Thanks for your help.

From: Thomas Mason
Sent: Thursday, September 7, 2017 10:08 AM
To: 'wakatime/wakadump' <reply@reply.github.com>; wakatime/wakadump <wakadump@noreply.github.com>
Cc: Author <author@noreply.github.com>
Subject: RE: [wakatime/wakadump] CSV export argument must be str not bytes (#8)

Yes, I got it to work yesterday sorry. I just changes the unicodecsv py3.py file to make it work.

Having said that…  the file that comes out is not very helpful.  Would it possible to output a file such as the attached? This would be much more useful as one could create pivot tables to group and dissect the data a lot better in this format.
